DependencyInjection/Configuration.php

```diff
@@ -72,6 +72,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('stream_factory')->defaultNull()->end()
                     ->end()
                 ->end()
+                ->booleanNode('toolbar')->defaultTrue()->end()
```

Comment:
Agreed. But I would like `canBeDisabled` instead... It should be enabled by default.